server/chat-commands.js

```diff
@@ -662,28 +662,16 @@ const commands = {
 		target = Chat.nicknamefilter(target, user, true);
```

Comment:
Unrelated to this change, but @Zarel, how did you come up with 32? You can fit 19 W's (widest character) before exceeding the user card.

Reply:
I vaguely remember talking about this, but it's possible it got lost in the internet trouble.

In any case, I think there's no need to restrict statuses to a single line, and I also think there's no need to optimize for the widest characters rather than average uses.

Reply:
Like, anyone literally having a status of `WWWWWWWWWWWW` is not going to be particularly concerned about readability.